README.md

```diff
@@ -1,2 +1,7 @@
 # hello-world
 Github tutorial
+<h1>All About Me</h1>
+<p>So you want to know about me? Well, I want to learn new things, all the time. Right now I'm trying to learn more about <i>Git-hub</i>. I guess that's not a <strong>big</strong> surprise, right?
+<br>
+This is the file that is being presented in the Github "Hello World" tutorial.<br>
+I can use mark-up language in this file to make things **bold** etc. There is a markdown cheatsheet linked in the tutorial in case I need it. 
```

Comment:
This might be an even better place to learn markdown language: [https://www.markdownguide.org](url)